src/main/resources/org/jenkinsci/plugins/workflow/job/console/NewNodeConsoleNote/script.js

```diff
@@ -1,62 +1,79 @@
-Behaviour.specify("span.pipeline-new-node", 'NewNodeConsoleNote', 0, function(e) {
-    if (e.processedNewNodeConsoleNote) {
-        return
-    }
-    e.processedNewNodeConsoleNote = true
-    var label = e.getAttribute('label')
-    if (label != null) {
-        var html = e.innerHTML
-        var suffix = ' (' + label.escapeHTML() + ')';
-        if (html.indexOf(suffix)<0) {
-            e.innerHTML = e.innerHTML.replace(/.+/, '$&' + suffix) // insert before EOL
+/**
+ * The setTimeout() method calls a function or evaluates an expression after a specified number of milliseconds.
+ * It is used to prevent a browser freeze that might occur if a very big log is being evaluated.
+ */
+setTimeout(() => {
```

Comment:
This smells wrong. I would expect the timeout to be applied to the callback, not to the call to `Behaviour.specify` itself.

Reply:
I do agree with that. Will change it.